src/Client/Client.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Why is there a client project even? Whats the point of this?
I looked at the pages/startup and couldn't figure it out.
Is this for testing?

Edit: If this is a further separation of responsibility I am not going to say anything but is it?
Edit2: if this is really just for testing to see if it works, remove it please. Make integration tests, not this. If anything this is an E2E test. Atleast move this to tests folder

Reply:
I added this because the project does not work with core without changing it's configuration.

The idea is that we have something that can be run without any complications. After codidact/core#61 is merged, I would add a pull request that changes the configuration to work with this project, and then remove this client entirely.

Reply:
Well considet that the configuration changes that are needed to be done are client secret changes or authority changes which I would expect anyone who downloads this code and deploys to do on their own. These configuration changes should be noted in the readme like the connection string. 

Since we are removing this either way in the future or now decide on your own which you want to do

Reply:
46cd8cf632b649c460b5b4d25cec5a25f94cfd88